### PR TITLE
Level200SummonsSpec

### PR DESCRIPTION
**Pull request by harliq**
Updated All level 200 summons to require Spec Summoning

### CODE REVIEW

File: Database/Patches/2013-02-BalanceOfPower/9 WeenieDefaults/PetDevice/48956 Fire Skeleton Samurai Essence (200).sql [hunk 1]
```diff
@@ -1,7 +1,7 @@
-DELETE FROM `weenie` WHERE `class_Id` = 48956;
+DELETE FROM `weenie` WHERE `class_Id` = 48956;
```
Comment (gmriggs) [inline Database/Patches/2013-02-BalanceOfPower/9 WeenieDefaults/PetDevice/48956 Fire Skeleton Samurai Essence (200).sql:1]: not sure which script is being used to generate these files, but might want to double-check it, as it seems to be adding new EOL spaces / blank lines at end of files

File: Database/Patches/2013-02-BalanceOfPower/9 WeenieDefaults/PetDevice/48956 Fire Skeleton Samurai Essence (200).sql [hunk 2]
```diff
@@ -22,6 +22,7 @@ VALUES (48956,   1,        128) /* ItemType - Misc */
      , (48956, 362,          2) /* SummoningMastery - Necromancer */
      , (48956, 366,         54) /* UseRequiresSkill */
      , (48956, 367,        570) /* UseRequiresSkillLevel */
+     , (48956, 368,         54) /* UseRequiresSkillSpec */
```
Comment (gmriggs) [inline Database/Patches/2013-02-BalanceOfPower/9 WeenieDefaults/PetDevice/48956 Fire Skeleton Samurai Essence (200).sql:25]: an associated shard update script will also be needed for this. currently everyone who has these items will have the versions that don't require summoning spec

Comment (harliq) [reply Database/Patches/2013-02-BalanceOfPower/9 WeenieDefaults/PetDevice/48956 Fire Skeleton Samurai Essence (200).sql:25]: Ripley is working on script

File: Database/Patches/2013-02-BalanceOfPower/9 WeenieDefaults/PetDevice/48956 Fire Skeleton Samurai Essence (200).sql (hunk 1 above)
Comment (harliq) [reply Database/Patches/2013-02-BalanceOfPower/9 WeenieDefaults/PetDevice/48956 Fire Skeleton Samurai Essence (200).sql:1]: I seen that.. its WeenieFab... Already fixed it.  Noticed it when I did the PR

**Pull request merged**